python/pyhie/allspark/model/deepseek_v3.py

```
@@ -83,86 +88,87 @@ def _build_graph(self, torch_cfg, derive_type):
+                "yarn_beta_fast": float(rope_scaling.get("beta_fast", 32)),
+                "yarn_beta_slow": float(rope_scaling.get("beta_slow", 1)),
+                "yarn_mscale": float(rope_scaling.get("mscale", 1.0)),
+                "yarn_mscale_all_dim": float(rope_scaling.get("mscale_all_dim", 1.0)),
             }
         self.rope_base = rope_base
 
         # Number of dense layers (first N layers use standard FFN, rest use MoE)
-        first_k_dense_replace = torch_cfg.get('first_k_dense_replace', 3)
-        moe_intermediate_size = torch_cfg.get('moe_intermediate_size', 2048)
+        first_k_dense_replace = torch_cfg.get("first_k_dense_replace", 3)
+        moe_intermediate_size = torch_cfg.get("moe_intermediate_size", 2048)
```

Flake8 (7.3.0)

[error] local variable 'moe_intermediate_size' is assigned to but never used

(F841)